src/components/vm/disks/diskAdd.tsx

```diff
@@ -337,13 +455,13 @@ const CustomPath = ({ idPrefix, onValueChanged, device, validationFailed, hideDe
                 <FileAutoComplete
                     id={`${idPrefix}-file-autocomplete`}
                     placeholder={_("Path to file on host's file system")}
-                    onChange={value => debounce(250, onValueChanged("file", value))}
+                    onChange={(value: string) => onValueChanged("file", value)}
                     superuser="try" />
                 <FormHelper fieldId={`${idPrefix}-file-autocomplete`} helperTextInvalid={validationFailed.customPath} />
             </FormGroup>
             {!hideDeviceRow && <FormGroup label={_("Device")}>
                 <FormSelect id={`${idPrefix}-select-device`}
-                        onChange={(_event, value) => onValueChanged('device', value)}
+                        onChange={(_event, value) => onValueChanged('device', value as "disk" | "cdrom")}
```

Comment:
Could this become a cockpit.assert() in the handler?

Reply:
Yes, why not.